.eslintrc.yaml

```diff
@@ -8,6 +7,29 @@ env:
   node: true
   es6: true
 rules:
+  "@typescript-eslint/consistent-type-assertions": warn
+  "@typescript-eslint/no-namespace": warn
+  "@typescript-eslint/no-var-requires": warn
+  no-unused-expressions: off
+  "@typescript-eslint/no-unused-expressions": warn
+  curly: warn
+  max-len:
+    - warn
+    - code: 150
```

Comment:
Q: 150 vs 120?
I'll be more than happy to increase the prettier printWidth to 150 🙏 
https://github.com/zowe/vscode-extension-for-zowe/blob/383e25cdfd6f5ee2bfb9b9e0a8c666fb47e1733f/prettier.config.js#L1